web-client/src/app/components/pay-from-to/pay-from-to.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-pay-from-to',
+  templateUrl: './pay-from-to.component.html',
+  styleUrls: ['./pay-from-to.component.scss'],
```

Comment:
is `styleUrls` compulsory? The `pay-from-to.component.scss` file is empty.

Reply:
Every component/page get a style file upon creation.
It doesn't really matter if it's empty or not because it's dealt with on build with the [tree shaking](https://developer.mozilla.org/en-US/docs/Glossary/Tree_shaking)